model-lowerings/src/org/jetbrains/dukat/commonLowerings/removeDuplicateMembers.kt

```diff
@@ -50,7 +59,14 @@ private class RemoveDuplicateMembersLowering : TopLevelModelLowering {
 }
 
 class RemoveDuplicateMembers : ModelLowering {
+    override fun lower(source: SourceSetModel): SourceSetModel {
+        return source.copy(sources = source.sources.map {
+            it.copy(root = RemoveDuplicateMembersLowering(ModelContext(source))
+                .lowerRoot(it.root, NodeOwner(it.root, null)))
+        })
+    }
+
     override fun lower(module: ModuleModel): ModuleModel {
```

Comment:
this looks redudnand to me (that is, we can leave only `lower(source: SourceSetModel): SourceSetModel {`

Reply:
We actually need to override this because it is not implemented in `ModuleModelLowering`